src/Components/MovieCard.js

```diff
@@ -6,8 +6,8 @@ const MovieCard = ({ title, poster, averageRating, releaseDate, id, backdrop, fa
   let navigate = useNavigate();
  
   return (
-    <article id ={id} className="movieCard" onClick={() => navigate(`/${id}`)}>
-      {<img src={poster} alt={ title + ` poster`} className='movieCardPoster'/>}
+    <article id ={id} className="movieCard">
+      {<img src={poster} alt={ title + ` poster`}  onClick={() => navigate(`/${id}`)} className='movieCardPoster'/>}
```

Comment:
Thank you for moving the event listener to the poster. This will help the user be able to click on the poster to view the movieDetails page directly instead of having it conflict with our favorite button.